### DISCUSSION TITLE
how can i make a popover not dismiss when clicked?

### DISCUSSION THREAD

**Discussion by machineonamission** [twbs/bootstrap#36562] [Q&A]
when using the "focus" trigger, the popover dismisses when clicked. when using the "click" trigger, it doesn't dismiss when clicked outside of. I want something like the focus trigger but it doesn't dismiss when clicked on (my popover has interactive elements). is there an easy way to do this or do I have to override it with the manual trigger?

**Comment by WindingSnow** [comment]
Probably overriding it with manual trigger should work.
